src/backend/crossterm.rs

```diff
@@ -8,12 +8,20 @@ use crossterm::error::ErrorKind;
 
 pub struct CrosstermBackend {
     screen: crossterm::Screen,
+    crossterm: crossterm::Crossterm,
+    // Need to keep the AlternateScreen around even when not using it directly,
+    // see https://github.com/TimonPost/crossterm/issues/88
```

Comment:
Even though I understand the issue, I doubt that the tui crate should cope explicitly with this issue. I would rather have the user aware of the limitations or specific behaviors of the backend she/he is using. Wouldn't a comment in the documentation be enough ?

Reply:
I just pushed a cleaned-up version which I think uses crossterm in the most coherent way, where all `Backend` methods only interact with the `crossterm::Crossterm` instance. This seems like a good idea to me independently of the alternate screen issue. 

Concerning the alternate screen: I agree that keeping the `AlternateScreen` instance around is something crossterm should ultimately deal with (I filed an issue [here](https://github.com/TimonPost/crossterm/issues/88)). However, before this is fixed on the crossterm side holding on to the `AlternateScreen` seems like a small price to pay for a working alternate screen mode. Alternatively, as you say the docs could mention that alternate screen mode is currently not implemented for the crossterm backend. It comes down to a trade-off between working around some issues in the backends and missing features.

Reply:
Ok presented that way you are right that it is a small price to pay. I will open an issue to not forget to revisit this problem if `crossterm` updates its behavior.